app/models/form/lettings/questions/uprn_selection.rb

```diff
@@ -31,4 +31,12 @@ def displayed_answer_options(log, user = nil)
   def hidden_in_check_answers?(log, _current_user = nil)
     (log.uprn_known == 1 || log.uprn_confirmed == 1) || !(1..10).cover?(log.address_options&.count)
   end
+
+  def input_playback(log = nil)
+    return unless log&.address_line1_input || log&.postcode_full_input
+
+    address_options_count = answer_options(log).count > 1 ? answer_options(log).count - 2 : 0
```

Comment:
-2 here cause there would always be a "none of the addresses" and "divider" options